packages/react-components/src/components/UploadBar/UploadBar.tsx

```diff
@@ -0,0 +1,168 @@
+import * as React from 'react';
+import cx from 'clsx';
+import {
+  Check as CheckIcon,
+  Error as ErrorIcon,
+  ChevronDown as ChevronDownIcon,
+  ChevronUp as ChevronUpIcon,
+} from '@livechat/design-system-icons/react/material';
+
+import {
+  FileUploadProgress,
+  FileUploadProgressActions,
+} from '../FileUploadProgress';
+import { ProgressStatus, ProgressSize } from '../Progress/constants';
+import { Icon } from '../Icon';
+import { ProgressCircle } from '../Progress';
+
+import styles from './UploadBar.module.scss';
+import { CSSTransition, TransitionGroup } from 'react-transition-group';
+
+const baseClass = 'upload-bar';
+const wrapperHeaderClass = `${baseClass}__wrapper__header`;
+
+export interface UploadBarProps {
+  className?: string;
+  percent: number;
+  title: string;
+  expanded?: boolean;
```

Comment:
What is the approach in DS v1 with boolean properties like this? in AA we assumed this form:

```suggestion
  isExpanded?: boolean;
```